kalamine/layout.py

```diff
@@ -174,7 +195,7 @@ def __init__(self, filepath: Path) -> None:
         self.meta["lastChange"] = datetime.date.today().isoformat()
 
         # keyboard layers: self.layers & self.dead_keys
-        rows = GEOMETRY[self.meta["geometry"]]["rows"]
+        rows = GEOMETRY[self.meta["geometry"]].rows
```

Comment:
That’s quite exactly what I wanted to see from these data classes.

I’ll try to use that for `meta` later today.